src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/representation/AddressDTO.java

```diff
@@ -1,22 +1,18 @@
 package uk.gov.ons.ctp.integration.contactcentresvc.representation;
 
-import java.io.Serializable;
-import lombok.AllArgsConstructor;
-import lombok.Builder;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 import lombok.Data;
-import lombok.NoArgsConstructor;
 
 @Data
-@Builder
-@NoArgsConstructor
-@AllArgsConstructor
-public class AddressDTO implements Serializable {
+@JsonPropertyOrder({"uprn", "formatted-address", "welsh-formatted-address"})
+public class AddressDTO {
 
   private String uprn;
 
+  @JsonProperty("formatted-address")
```

Comment:
Sorry - you missed the memo. kebab-case abandoned in favour of camelCase

Reply:
Changed to camel case